modules/local/format_taxresults_kraken2.nf

```diff
@@ -0,0 +1,107 @@
+process FORMAT_TAXRESULTS_KRAKEN2 {
+    label 'process_low'
+
+    conda "bioconda::bioconductor-dada2=1.22.0 conda-forge::r-digest=0.6.30"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bioconductor-dada2:1.22.0--r41h399db7b_0' :
+        'biocontainers/bioconductor-dada2:1.22.0--r41h399db7b_0' }" // TODO: pure R seems sufficient!
```

Comment:
what about this todo?

Reply:
Good catch! Forgot about it. I check if I can use R only.

Reply:
R-only was successful, changed it, thanks for spotting it!